app/helpers/media_helper.rb

```diff
@@ -1,13 +1,11 @@
 module MediaHelper
   def media_thumbnail(media_item)
-    if media_item.thumbnail?
-      if media_item.caption.text.present?
-        alt_text = truncate(media_item.caption.text, length: 36)
-        return image_tag(media_item.item.thumb.url, alt: alt_text)
-      else
-        # Use auto-generated alt text
-        return image_tag(media_item.item.thumb.url)
-      end
+    if media_item.thumbnail? && media_item.caption
+      # Use auto-generated alt text if no caption text
+      return image_tag(media_item.item.thumb.url) unless media_item.caption.text.blank?
+
+      alt_text = truncate(media_item.caption.text, length: 36)
+      image_tag(media_item.item.thumb.url, alt: alt_text)
     else
       content_tag(:div, class: "media-block") do
```

Comment:
The else refers to other types of mdia that are not images. So we do want to capture all images (with and without captions) in the first part of the conditional.

Reply:
There's different icons for other types of media not images. You can ignore this part.

But any images you test should be displaying as images and not other types of media.